### ISSUE TITLE
[FEAT] Metadata support for the Supabase data provider

### ISSUE THREAD

**Issue by fspijkerman** [pankod/refine#1489]
Allow overriding `select('*')` via metadata in the Supabase data provider.

Example of how you can use it:
```
const { tableProps } = useTable<IEvent>({
  permanentFilter: [{ field: "devices.tenant_id", operator: "eq", value: tenant }],
  metaData: {
    select: '*, devices!inner(tenant_id)'
  },
});
```

**Comment by omeraplak** [comment]
Hey @fspijkerman ,
Amazing, Thank you! I'll release it soon

**Comment by omeraplak** [comment]
Hey @fspijkerman ,
We've released `@pankod/refine-supabase@2.7.5`. Thank you!
